import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/monitoring/Event.java

```diff
@@ -21,6 +21,8 @@
   CSTORE_ERROR(Constants.prefix + "cstore_errors"),
   CSTORE_BACKUP_ERROR(Constants.prefix + "cstore_backup_errors"),
   CSTORE_BYTES(Constants.prefix + "cstore_bytes"),
+  CSTORE_409_ERROR(Constants.prefix + "cstore_409_error"),
+  CSTORE_5xx_ERROR(Constants.prefix + "cstore_5xx_error"),
```

Comment:
We already have monitoring codes for CSTORE_ERROR, adding a metric for every status code might make this hard to configure alerts on. Can we just increment the regular CSTORE_ERROR metric for all additional codes?